### ISSUE TITLE
PRDT: Determine and fill out gaps in MVD

### ISSUE THREAD

**Issue by meyerdarcie** [bcgov/reserve-rec-api#9]
#### Description:

This ticket is determine requirements and criteria for our minimum viable dataset for a backcountry prototype, as referenced in #6 

#### Acceptance Criteria:
[Note: Use 'Given/When/Then' format if it makes sense to. Otherwise, a simple checklist that can be tested.]

#### Development Checklist:

- [ ] ...
- [ ] ...
- [ ] ...

#### Dependencies

- Blocked by
- Blocking

**Relevant documentation as reference**

**Definition of Ready**

- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if applicable)
- [ ] Design / Solution is accepted by Product Owner (if applicable)
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

**Definition of Done** 
- In progress:
  - [ ] Acceptance criteria are tested (Functionality meets the acceptance criteria defined in the ticket)
  - [ ] UI meets accessibility requirements
  - [ ] Unit tests are written
  - [ ] Work is traceable in GitHub
  - [ ] PR linked to ticket number
  - [ ] If needed/required - Dev adds flag/label to highlight any migration steps necessary prior to PROD deployment
- Code review:
  - [ ] Code is peer reviewed and has passed CI/CD tests
- QA:
  - [ ] Acceptance criteria are tested (Functionality meets the acceptance criteria defined in the ticket)
  - [ ] Code is potentially shippable to the production environment
  - [ ] Functional features have been tested and passed by QA
  - [ ] UI components tested by designer
  - [ ] Code is deployed to PROD when moved to 'done' column (unless requested otherwise by PO)
- PO Review:
  - [ ] Acceptance criteria are tested (Functionality meets the acceptance criteria defined in the ticket)
  - [ ] Reviewed and approved by Product Owner

#### Notes:
- 
-

**Comment by Dianadec** [comment]
Qs for refinement: What are the gaps, how will we find/determine that? What is this ticket about

**Comment by Dianadec** [comment]
Todo: get more information re: Strapi and work with Rob
Some info in this teams thread: https://teams.microsoft.com/l/message/19:24137c6e9eba4bb7b32a82fe6dd469b9@thread.tacv2/1719956073977?tenantId=6fdb5200-3d0d-4a8a-b036-d3685e359adc&groupId=cb2c4bec-527d-4097-80b2-6e40e98d36bf&parentMessageId=1719956073977&teamName=Parks%20%26%20Recreation%20Transformation%20(Open%20X-Gov)&channelName=%5BTeam%5D%20PRDT%20Internal%20Collab%20(Closed)&createdTime=1719956073977&ngc=true

**Comment by Dianadec** [comment]
Also need: one Source of truth for dates - Amanda from oxd has a mural board.

another todo: generate list of current dataset info and get feedback on it

**Comment by Dianadec** [comment]
DR sub areas research findings: https://bcgov.sharepoint.com/:w:/r/teams/031072/_layouts/15/Doc.aspx?sourcedoc=%7BAF8DE1B3-F20B-4153-872B-97CEC07668FF%7D&file=DR%20sub%20areas%20research%20findings.docx&wdOrigin=TEAMS-MAGLEV.p2p_ns.rwc&action=default&mobileredirect=true

**Comment by Dianadec** [comment]
Refinement notes:
- we do have the raw data of the initial several parks
- but geospatial isn't in the right spot, but they are pretty good
- In the current format, assume it's perfect

**Comment by Dianadec** [comment]
Going to create a new ticket that is a more focused version of this one.

**Comment by meredithom** [comment]
Refinement notes:
- To explore further: How data updates are going to happen and how that will be communicated to regional staff 
- We should be keeping track of data sources as we go in [the spreadsheet](https://bcgov.sharepoint.com/:x:/r/teams/08548/_layouts/15/doc2.aspx?sourcedoc=%7B97E683BA-1C34-4F8C-8C3A-ABFCF23C569D%7D&file=Data%20Collection%20Backcountry%20-%20Cape%20Scott-JDF-Strathcona.xlsx&action=default&mobileredirect=true) 
- DR will be our source of truth for protected area and subarea names
 
- DB keeps track of last updated dates of data and surface in admin side of application
- Start with parks that have the most complete data (best data will be from parks with inventory)

Todo:
- diagram of our process around gathering data

**Comment by Dianadec** [comment]
> [@cameronpettit](https://github.com/cameronpettit) is this kind of already part of what you're doing with the first 3 parks? Thinking this might be old or duplicated.

@cameronpettit following up - should we mark this as a dupe?

**Comment by Dianadec** [comment]
Oct 16 refinement notes:
- this is for all tickets, not just the first 3.
- want to validate the date from first 3 parks prior to starting the rest of the backcountry parks.

Pulling this one out until we are ready to look at the additional parks.

When/how will we validate the first 3?
- Not until we can show things in the front end and how well that works.

**Comment by cameronpettit** [comment]
Captured by the work I am doing with Jess M. I can close if needed.

**Comment by LindsayMacfarlane** [comment]
August 13, 2026 - closing ticket. Much work has been done since this ticket was created. It is now outdated and a duplicated.
